spiketoolkit/comparison/sortingcomparison.py

```diff
@@ -5,7 +5,7 @@
 import pandas as pd
 
 from .comparisontools import (count_matching_events, compute_agreement_score,
-                                                do_matching, do_counting, do_confusion_matrix)
+                                                do_matching, do_score_labels, do_counting, do_confusion_matrix)
 
 
```

Comment:
@samuelgarcia could you make a function `compare_sorters` that returns a `SortingComparison` object?
I think that it is in line with the preprocessing module and easier to use. Then we only have to add docstring for the function.